charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -43,6 +43,34 @@
         }
       }
     },
+    "DeleteAssetRequest": {
+      "type": "object",
+      "required": [
+        "destinationCatalogID"
+      ],
```

Comment:
destinationAssetID is required as well

Reply:
Yes, it is exactly the same one. No need for destinationCatalogID.

Reply:
Fixed. Thanks.